SPECS/xcp-clipboardd.spec

```diff
@@ -0,0 +1,38 @@
+Name:           xcp-clipboardd
+Version:        1.0.2
+Release:        1{%dist}
```

Comment:
%{dist}

Reply:
Copy paste of your previous comment. :wink: (Copy & Paste is bad yes. ^^)